.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,23 @@
 ### Description
 
-<!-- Please explain the changes you made here. -->
+<!--
+Please explain the changes you made here.
+
+Help your reviewers my guiding them through your key changes,
+implementation decisions etc.
+You can even include snippets of output or screenshots.
+
+A good description == a faster review :)
```

Comment:
Minor: "A good clear description". Good is very subjective. Clear isn't much better but it helps that they need to be clear in what they're doing.